README.md

```diff
@@ -144,3 +144,17 @@ eval $(gds aws security-test -e); make e2e
 The tests are automatically run against staging in the 
 pipeline but can be run manually by assuming a 
 different role and setting different credentials. 
+
+## Configuration
+
+The config.py module provides an interface to the  
+standard Flask app.config. The reason for this is 
```

Comment:
_The interface makes it easier and more effective to set and get configuration values. We retrieve configuration settings without having to import the app every time by using config functions in modules which don't have access to the flask app object._

Is this more succinct?